### PR TITLE
Reorganizing data sent to recycler asset list

### PR DESCRIPTION
**Pull request by jinchung**

### CODE REVIEW

**Comment by jinchung** [comment]
Tested wallet screen and send flow all work as expected

**Comment by jinchung** [comment]
IMG_2476
Space between sections also taken care of

**Comment by jinchung** [comment]
[Spacing / padding on Wallet Screen for RecyclerActivityList should be 42](https://trello.com/c/Gpd0UVD0/100-spacing-padding-on-wallet-screen-for-recycleractivitylist-should-be-42)

**Comment by jinchung** [comment]
[Clean up sections data used for Recycler Asset List](https://trello.com/c/pxKlxF9g/103-clean-up-sections-data-used-for-recycler-asset-list)

File: src/components/asset-list/RecyclerAssetList.js [hunk 1]
```diff
@@ -25,7 +26,7 @@ const Wrapper = styled.View`
 `;
 
 // eslint-disable-next-line react/prop-types
-const AssetListHeaderRenderer = ({ section }) => <AssetListHeader {...section} />;
+const AssetListHeaderRenderer = (data) => <AssetListHeader {...data} />;
```
Comment (osdnk) [inline src/components/asset-list/RecyclerAssetList.js]: 😅

**Pull request merged**